pydemolib/nessiedemo/demo.py

```diff
@@ -131,6 +132,16 @@ def get_iceberg_version(self: T) -> str:
         """Get the Iceberg version defined in the versions-dictionary."""
         return self.__iceberg_version
 
+    def get_iceberg_download_url_for_jar(self: T, iceberg_component: str) -> str:
+        """Get the Iceberg download URL for a jar file of the given iceberg component."""
+        version = self.get_iceberg_version()
+
+        home = Path.home()
```

Comment:
What's the benefit of `Path.home()` over `os.environ["HOME"]`?

Reply:
fixed